readme and screenshot (#66)
nextcloud/photos: README.md

> # Photos

Reviewer: ```suggestion
# Nextcloud Photos
```
That’s a bit nicer here :)

**📸 Your memories under your control**

![](screenshots/screenshot1.png)

## Why is this so awesome?
-   **📸 Beautiful Photo und Video Timeline**
-   **⭐ Favorites and Tagging:** Favorite and tag your photos!
-   **Slideshow and easy sharing** Show slideshows and share your photos or albums easy

> -   **Albums:** Create albums from your contents

Reviewer: Indentation here is also a bit much, just needs to be 1 space after the dash.

    Future plans:
-   

## 🚀 Installation  

> In your Nextcloud, simply navigate to »Apps«, find the Photos app and enable it. Then open the app from the app menu.Consider installing the preview generator for pregenerating the thumnailsPlease have a look at [CHANGELOG.md](https://github.com/nextcloud/photos/blob/master/CHANGELOG.md) for details about changes.

Reviewer: Some missing spaces and dots at end of sentences here in the middle. :)

Reviewer: Also would be good to link the preview generator here? :)

    
## Mobile Photos
Use the Android/iOS App to view your photos. It's possible to autoupload them.

##Maintainers:
- [John Molakvoæ](https://github.com/skjnldsv)
If you'd like to join, just go through the [issue list](https://github.com/nextcloud/photots/issues) and fix some. :)
    
## 🏗 Development setup
    1.  ☁ Clone this into your `apps` folder of your Nextcloud
    2.  👩‍💻 In a terminal, run the command `make dev-setup` to install the dependencies
    3.  🏗 Then to build the Javascript whenever you make changes, run `make build-js`
    4.  ✅ Enable the app through the app management of your Nextcloud

>     5.  🎉 Partytime!

Reviewer: These don’t need to be indented, right?

# Nextcloud Photos
**📸 Your memories under your control**

![](screenshots/screenshot1.png)

## Why is this so awesome?
- **📸 Beautiful Photo und Video Timeline**
- **⭐ Favorites and Tagging:** Favorite and tag your photos!
- **Slideshow and easy sharing** Show slideshows and share your photos or albums easy
- **Albums:** Create albums from your contents
    
    Future plans:
- ...

## 🚀 Installation  

> In your Nextcloud, simply navigate to »Apps«, find the Photos app and enable it. Then open the app from the app menu. Consider installing the [preview generator](https://github.com/rullzer/previewgenerator) for pregenerating the thumnails. 

Reviewer 2: Should be `« Apps »`

Reviewer: That’s only the french way. ;)

Let’s just go for:

> In your Nextcloud, go to the Apps management and enable the Photos app.

Reviewer 2: Fine ^^

> In your Nextcloud, simply navigate to »Apps«, find the Photos app and enable it. Then open the app from the app menu. Consider installing the [preview generator](https://github.com/rullzer/previewgenerator) for pregenerating the thumnails. 

Reviewer 2: `thumnails` > `thumbnails`

Please have a look at [CHANGELOG.md](https://github.com/nextcloud/photos/blob/master/CHANGELOG.md) for details about changes.
    
## Mobile Photos

> Use the Android/iOS App to view your photos. It's possible to autoupload them.

Reviewer 2: `auto-upload`

##Maintainers:
- [John Molakvoæ](https://github.com/skjnldsv)

> If you'd like to join, just go through the [issue list](https://github.com/nextcloud/photots/issues) and fix some. :)

Reviewer 2: wrong link >> `https://github.com/nextcloud/photos/issues`

> ## 🏗 Development setup

Reviewer 2: Would you mind adding an explanation for the linters? 
So everyone can lint and easily fix linting issues.
Also for vscode user we can add some tips for extensions.

Author: Do you mean adding the command "npm run lint:fix"?

Reviewer 2: Yep

1.  ☁ Clone this into your `apps` folder of your Nextcloud
2.  👩‍💻 In a terminal, run the command `make dev-setup` to install the dependencies

> 3.  🏗 Then to build the Javascript whenever you make changes, run `make build-js`

Reviewer 2: To watch, the command is `make watch-js`

# Nextcloud Photos
**📸 Your memories under your control**

![](screenshots/screenshot1.png)

## Why is this so awesome?
- **📸 Beautiful Photo und Video Timeline**
- **⭐ Favorites and Tagging:** Favorite and tag your photos!
- **Slideshow and easy sharing** Show slideshows and share your photos or albums easy
- **Albums:** Create albums from your contents
    
    Future plans:
- ...

## 🚀 Installation  

> In your Nextcloud, simply navigate to »Apps«, find the Photos app and enable it. Then open the app from the app menu. Consider installing the [preview generator](https://github.com/rullzer/previewgenerator) for pregenerating the thumbnails. 

Reviewer 2: »Apps« still not fixed should be `Apps`

Reviewer: ```suggestion
In your Nextcloud, simply enable the Photos app through the Apps management.

Consider installing the [preview generator](https://github.com/rullzer/previewgenerator) for pregenerating thumbnails. 
```
